doc/CI_Hardware.md

```diff
@@ -0,0 +1,221 @@
+# Tock Hardware-in-the-Loop CI
+In addition to software tests, Tock allows platform maintainers to deploy CI runners for hardware platforms, to ensure continuous on-metal performance correctness. The hardware CI is federated, and maintainers for each hardware platform are responsible for ensuring that their hardware CI system is set up, accessible, and runs correctly.
+
+The Tock core team maintains the following hardware CI instances:
+- Nrf52840dk	UC San Diego; contact: Pat Pannuto <ppannuto@ucsd.edu>, Anthony Quiroga <aquiroga@ucsd.edu> 
+- Hail (WIP)	UC San Diego; contact: Pat Pannuto <ppannuto@ucsd.edu>, Anthony Quiroga <aquiroga@usd.edu>
+
+<!-- npm i -g markdown-toc; markdown-toc -i CI_Hardware.md -->
```

Comment:
This comment is actually a command you can/should run in your terminal command line.

`npm` is the Node package manager -- if you don't have node installed, you can follow the directions here to install node: https://nodejs.org/en/download/

once this runs, an actual table of contents should show up between `toc` and `tocstop`